frontend/src/Login.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+<div>
+    <div class="columns is-centered">
+        <div class="column is-size-4">Please login</div>
+    </div>
+    <div class="columns is-centered">
+        <form class="column is-narrow login" @submit.prevent="login">
+            <div v-if="!this.$store.getters.isAuthenticated">
+                <b-field>
+                    <b-input
+                        placeholder="Name"
+                        minlength="1"
+                        type="text"
+                        required
+                        v-model="name"
+                    ></b-input>
+                </b-field>
+                <b-field>
+                    <b-input
+                        placeholder="Password"
+                        minlength="1"
+                        type="password"
+                        required
+                        v-model="password"
+                    ></b-input>
+                </b-field>
+                <b-button
+                    :disabled="missingInput"
+                    type="submit"
+                    class="button is-fullwidth"
+                    @click="login()"
+                >Login</b-button>
+            </div>
+        </form>
+    </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator';
+import { getModule } from 'vuex-module-decorators';
+import { mapGetters } from 'vuex';
+import AuthenticationStatus from './authentication/AuthenticationStatus';
+import AuthenticationStore from './authentication/AuthenticationStore';
+
+@Component({
+    computed: mapGetters(['authStatus'])
+})
+export default class Login extends Vue {
+    name: string = '';
+    password: string = '';
+
+    authenticationStore = getModule(AuthenticationStore);
+
+    get missingInput() {
+        return this.name.length === 0 || this.password.length === 0;
+    }
+
+    login() {
+        const { name } = this;
```

Comment:
Das her ist kein Problem, mich würde nur mal interessieren, was das für eine Syntax ist?

Reply:
Im Grunde ist es synonym zu "const name = this.name;", ich habe es mal geändert.

Reply:
Ah, hatte hier auf dem Rechner eslint noch nicht konfiguriert. Ich bekomme da die Warnung: Use object destructuring.eslint(prefer-destructuring).